tool/tsh/common/putty_config_windows.go

```diff
@@ -207,6 +237,16 @@ func addHostCAPublicKey(registryHostCAStruct puttyhosts.HostCAPublicKeyForRegist
 		return trace.Wrap(err)
 	}
 
+	// if matchHosts has any entries, delete the "MatchHosts" key from the registry as its entries were migrated above.
+	if len(matchHosts) > 0 {
+		log.Debugf("Deleting %v legacy MatchHosts value(s) from registry key %v", len(matchHosts), registryKeyName)
+		err := registry.DeleteValueFromRegistryKey(registryKey, "MatchHosts")
```

Comment:
All that `DeleteValueFromRegistryKey` does is call `registryKey.DeleteValue()` and log if there's an error. This code already logs if there's an error, so I don't see what we gain by adding an extra func.

Can we eliminate the duplicate logging and get rid of an exported func by removing `DeleteValueFromRegistryKey` here?